systest/cluster/cluster.go

```diff
@@ -263,6 +273,17 @@ func (c *Cluster) AddSmeshers(cctx *testcontext.Context, n int) error {
 	return nil
 }
 
+// DeletePoet delete all poet servers.
+func (c *Cluster) DeletePoet(cctx *testcontext.Context) error {
+	for _, pod := range c.poets {
+		if err := deletePoet(cctx, pod.Name); err != nil {
+			return err
+		}
+		c.poets = c.poets[1:]
```

Comment:
it will be using same slice, so if there are appends somewhere this will be a memory leak

Reply:
k. set it to nil at the beginning